lua/cmp_tabnine/source.lua

```diff
@@ -248,7 +286,10 @@ Source._on_stdout = function(_, data, _)
 	--
 	-- now, if we have a callback, send results
 	if Source.callback then
-		Source.callback(items)
+		Source.callback({
+			items = items,
+			isIncomplete = true,
```

Comment:
why set `isIncomplete` here?

Reply:
without it complete function of tn source will only be called once as you typed the first char of keyword. Later on even if you typed more chars for the same keyword, the new info won't be sent to tn to get updated completion because completion client regards the previously provided completion item list of tn is already *complete*.

But i'd like to mention that setting this will consistently send new info to tn on more keystrokes, which could be a performance break. May be an option for it?

Reply:
Yes, this should be a configurable option. 
Please add to the config object, set to `true` by default.
Please also add a line to README